MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -89,6 +100,54 @@ public class AudioFileProber
                 Fetch(item, result, cancellationToken);
             }
 
+            var libraryOptions = _libraryManager.GetLibraryOptions(item);
+
+            if (libraryOptions.EnableLUFSScan)
+            {
+                string output;
+                using (var process = new Process()
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = _mediaEncoder.EncoderPath,
+                        Arguments = $"-hide_banner -i \"{path}\" -af ebur128=framelog=verbose -f null -",
+                        RedirectStandardOutput = false,
+                        RedirectStandardError = true
+                    },
+                })
+                {
+                    try
+                    {
+                        process.Start();
+                    }
+                    catch (Exception ex)
+                    {
+                        _logger.LogError(ex, "Error starting ffmpeg");
+
+                        throw;
+                    }
+
+                    output = await process.StandardError.ReadToEndAsync(cancellationToken).ConfigureAwait(false);
+                    cancellationToken.ThrowIfCancellationRequested();
+                    MatchCollection split = Regex.Matches(output, @"I:\s+(.*?)\s+LUFS");
+
+                    if (split.Count != 0)
+                    {
+                        item.LUFS = float.Parse(split[0].Groups[1].ToString(), CultureInfo.InvariantCulture.NumberFormat);
```

Comment:
```suggestion
                        item.LUFS = float.Parse(split[0].Groups[1].ValueSpan, CultureInfo.InvariantCulture.NumberFormat);
```
This should work and prevent a string allocation, no?

Reply:
Sure, quickly tried that and works fine if you think that'll be better :)